specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.5-preview.1/backuprestore.json

```diff
@@ -82,6 +82,61 @@
         }
       }
     },
+    "/managedidentitymodebackup": {
+      "post": {
+        "tags": [
+          "ManagedIdentityModeFullBackup"
+        ],
+        "operationId": "ManagedIdentityModeFullBackup",
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        },
+        "description": "Creates a full backup using a user-provided Azure blob storage container uri.",
+        "parameters": [
+          {
+            "in": "body",
+            "name": "azureStorageBlobContainerUri",
+            "schema": {
+              "type": "string"
```

Comment:
The body should always be a model object. You can't expand this, so in the future if you need to add properties it's a breaking change. See https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#http-post-action-returns-200

Reply:
In addition to that, do you want to include an optional folder path parameter?